### PR TITLE
adding devdojo tails project

### PR DESCRIPTION
**Pull request by ganapathi12**
this project is very friendly for the first time contributors all you need to is add new components in tailwind css.
let me know if there's any feedback about the template.

### CODE REVIEW

**Comment by ganapathi12** [comment]
i am sorry to say this, but in the hactoberfest page i see that contributing to this repositery is ineligible (Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate.), so could you please add "hacktoberfest" topic to your repository. @ritwik12 @kudoabhijeet @KirankumarAmbati @floydpink @luckyguy73

File: _data/projects/Devdojo_tails.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+site: https://github.com/thedevdojo/tails
+tags:
+  - web
+  - javascript
+  - css
+  - education
+  - learning
+upforgrabs:
+  name: help wanted
+  link: https://github.com/thedevdojo/tails/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/Devdojo_tails.yml]: @ganapathi12 Please use this instead https://github.com/thedevdojo/tails/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
> i am sorry to say this, but in the hactoberfest page i see that contributing to this repositery is ineligible (Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate.), so could you please add "hacktoberfest" topic to your repository. @ritwik12 @kudoabhijeet @KirankumarAmbati @floydpink @luckyguy73

This is done!!

**Comment by ganapathi12** [comment]
Hi @ritwik12! the change you have suggested is done!

**Comment by ganapathi12** [comment]
> > i am sorry to say this, but in the hactoberfest page i see that contributing to this repositery is ineligible (Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate.), so could you please add "hacktoberfest" topic to your repository. @ritwik12 @kudoabhijeet @KirankumarAmbati @floydpink @luckyguy73
> 
> This is done!!

And could you please add "hacktoberfest-accepted" label to my PR so that i can actively participate in hacktober fest! @ritwik12

**Review by ganapathi12 — COMMENTED** [review]
Changed the requested changes!

File: _data/projects/Devdojo_tails.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: tails
+desc: Tails is a (no-config) copy'n paste library of hand-crafted templates and components built in TailwindCSS.
+      It's pretty simple to use any of these components or templates in your project. Any component you wish to use can be copied and pasted into your own project. Visit the Components page to view the components and view the source code. Additionally, you can view the Templates and simply copy and paste these templates into your own project.
```
Comment (ritwik12) [inline _data/projects/Devdojo_tails.yml]: @ganapathi12 I think we can remove this line to make the desc short.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
> > > i am sorry to say this, but in the hactoberfest page i see that contributing to this repositery is ineligible (Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate.), so could you please add "hacktoberfest" topic to your repository. @ritwik12 @kudoabhijeet @KirankumarAmbati @floydpink @luckyguy73
> > 
> > 
> > This is done!!
> 
> And could you please add "hacktoberfest-accepted" label to my PR so that i can actively participate in hacktober fest! @ritwik12

@ganapathi12 You would not need that as it will be merged after your corrections.

**Review by ganapathi12 — COMMENTED** [review]
@ritwik12 , updated requested changes!

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@ganapathi12 Thanks!!

**Pull request merged**